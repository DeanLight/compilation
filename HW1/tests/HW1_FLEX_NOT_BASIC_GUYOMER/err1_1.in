
% s
( s )
stream
s
endstream

s
