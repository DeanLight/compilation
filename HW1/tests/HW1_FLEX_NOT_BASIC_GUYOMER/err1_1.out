2 COMMENT % s
3 STRING  s 
6 STREAM s
Error s
